src/content/docs/style-guide/images/screenshots-images.mdx

```diff
@@ -144,8 +145,10 @@ We have four major categories of images on our site.
     Keep in mind New Relic Security guidelines and avoid private information from New Relic or from customers. In addition:
 
     * Always crop or mask sensitive information.
-    * Annotate images sparingly
-    * Consider adding a caption (see our [tips](#captions) below).
+    * Annotate images sparingly.
+    * Consider adding a caption. See our [tips](/docs/style-guide/images/screenshots-images/#captions) below.
+    * Avoid having non-UI elements like the mouse pointer or the browser's scrollbar.
+    * If you need to blur part of the screenshot, use the gaussian blur. 
```

Comment:
```suggestion
    * If you need to blur part of the screenshot, use the blur tool in Snagit. 
```
Consider adding an image here^

Reply:
@a-sassman, I think that the gaussian blur was indicated by @austin-schaefer. Austin, can you confirm this? or maybe I've made it up, I can't remember now...

Reply:
@nbaenam I'm seeing eight blur options in SnagIt. In the past we've used the first one. Could one of these be gaussian?